scripts/manage_study.py

```diff
@@ -266,116 +290,123 @@ def login(manager=None, dry_run=False, api_base=None, verbose=False):
 origin = env_origin_map[parsed_args.environment]
 api_base = origin + '/single_cell/api/v1/'
 
-# Login connection
-connection = login(manager=None, dry_run=parsed_args.dry_run, api_base=api_base, verbose=verbose)
-
-## Handle list studies
-if hasattr(parsed_args, "summarize_list"):
-    if verbose: print("START LIST STUDIES")
-    ret = connection.get_studies(dry_run=parsed_args.dry_run)
-    manage_call_return(ret)
-    print("You have access to "+str(len(ret[scp_api.c_STUDIES_RET_KEY]))+" studies.")
-    if not parsed_args.summarize_list:
-        print(os.linesep.join(ret[scp_api.c_STUDIES_RET_KEY]))
-
-## Create new study
-if hasattr(parsed_args, "study_description") and not parsed_args.study_name is None:
-    if verbose: print("START CREATE STUDY")
-    is_public = not parsed_args.is_private
-    ret = connection.create_study(study_name=parsed_args.study_name,
-                                  study_description=parsed_args.study_description,
-                                  branding=parsed_args.branding,
-                                  billing=parsed_args.billing,
-                                  is_public=is_public,
-                                  dry_run=parsed_args.dry_run)
-    manage_call_return(ret)
-    if succeeded(ret):
-        print('Created study')
-
-## Share with user
-if hasattr(parsed_args, "permission"):
-    if verbose: print("START SET PERMISSION")
-    ret = connection.set_permission(study_name=parsed_args.study_name,
-                                    email=parsed_args.email,
-                                    access=parsed_args.permission,
+if __name__ == '__main__':
+    # Login connection
+    connection = login(manager=None, dry_run=parsed_args.dry_run, api_base=api_base, verbose=verbose)
+
+    ## Handle list studies
+    if hasattr(parsed_args, "summarize_list"):
+        if verbose: print("START LIST STUDIES")
+        ret = connection.get_studies(dry_run=parsed_args.dry_run)
```

Comment:
Why did you name this ret?

Reply:
`ret` means "returned object".  That name and much of the other code in `manage_study.py` and `scp_api.py` were written by Tim.